src/Nethermind/Nethermind.Consensus/Processing/RecoverSignature.cs

```diff
@@ -87,7 +87,7 @@ public void RecoverData(Block block)
                     recoverFromEcdsa++;
                 }
 
-                if (tx.HasAuthorizationList)
+                if (poolTx is not null && tx.HasAuthorizationList)
```

Comment:
still increment recoverFromEcdsa++; ?

Reply:
We need to trigger the recovery if an authority is missing. We could add
`                        else if(tx.AuthorizationList[i].Authority is null)
`
maybe?